src/BUILD.gn

```diff
@@ -63,4 +63,12 @@ if (chip_build_tests) {
       deps += [ "${chip_root}/src/lib/shell/tests" ]
     }
   }
+
+  if (current_os != "zephyr") {
```

Comment:
`current_os == "linux"` right?

Reply:
You are right, happy only supports linux.